Inventory.cs

```diff
@@ -8,31 +8,169 @@ namespace JanasInventoryManagementSystem
 {
     public class Inventory
     {
-        private List<Product>? products = new List<Product>();
+        private static List<Product>? products = new List<Product>();
 
-        public void Add(Inventory inventory)
+        public void Add()
         {
+            Console.Write("Enter product name: ");
+            string name = Console.ReadLine();
 
+            Console.Write("Enter product price: ");
+            decimal price = Convert.ToDecimal(Console.ReadLine());
+
+            Console.Write("Enter product quantity: ");
+            int quantity = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine();
+
+            Product product = new Product(name, price, quantity);
+            products.Add(product);
+
+            Console.WriteLine("Product Added Successfully!\n");
+            Console.WriteLine("--------------------------------------\n");
         }
 
-        public void View(Inventory inventory) 
+        public void View() 
         {
-        
+            if (products.Count == 0)
+            {
+                Console.WriteLine("No Products Found!\n");
+                Console.WriteLine("--------------------------------------\n");
+            }
+            else
+            {
+                Console.WriteLine($"Products List:");
+
+                foreach (Product product in products)
+                {
+                    Console.WriteLine($"{product.Id}. {product.Name}\n" +
+                        $"Quantity: {product.Quantity}\n" +
+                        $"Price: {product.Price}\n");
+                }
+
+                Console.WriteLine("--------------------------------------\n");
+
+            }
         }
 
-        public void Delete(Inventory inventory)
+        public void Edit()
         {
 
+            if (products.Count == 0)
+            {
+                Console.WriteLine("No Products to Edit!\n");
+                Console.WriteLine("--------------------------------------\n");
+            }
+            else
+            {
+                Console.Write("Enter The Product Name to Be Edited: ");
+                string name = Console.ReadLine();
+
+                var product = GetProduct(name);
+
+                Console.WriteLine();
+
+                if (product != null && product.GetType() == typeof(Product))
```

Comment:
product != null expression is always true ,
please check  GetProduct(name) output when name does not exist

Reply:
why did we choose to check the type here 
product.GetType() == typeof(Product)
aren't we sure about it ?